src/pybind/mgr/dashboard/controllers/tcmu_iscsi.py

```diff
@@ -12,6 +12,8 @@
 @AuthRequired()
 class TcmuIscsi(RESTController):
     def _get_rate(self, daemon_type, daemon_name, stat):
+        # Also see ..services.ceph_service.CephService#get_rate
```

Comment:
is this comment relevant for the implementation of this function? if not I would remove it.

Reply:
This function and `.services.ceph_service.CephService#get_rate` are very similar and share a common history. In contrast to `.services.ceph_service.CephService#get_rate`, this function calculates the differentiation for all inputs, instead of just the last one. In the long term, we should IMO aim for unifying both functions. E.g. using this `get_rate` also in other places.

Reply:
Ok, in that case I would remove the comment, and open an issue in tracker.ceph.com to keep track of that enhancement.

Reply:
With this comment, the relationship between both `get_rate`s is also clear in the git history. Why do you think this comment is harmful?

Reply:
I don't think it's harmful, (it's just a comment), but usually comments explain or alert to some part of the code that is unclear, while this comment feels more like a "TODO" item, that should be tracked in the tracker rather than in the code.

Reply:
Note that I had changed this function after I noticed “get_rate” is broken throughout the dashboard [1]

[1] https://tracker.ceph.com/issues/23389